src/fingerprinting/hooks.ts

```diff
@@ -29,14 +30,17 @@ export function createFingerprintPreLaunchHook(browserPool: BrowserPool<any, any
         }
 
         launchContext.extend({ fingerprint });
+        // hide webdriver with arg.
+        // IMHO we could do this by default not only with fingerprinting.
```

Comment:
Yeah, agreed. I don't think it ever makes sense to run with webdriver: true

Reply:
we just have to be careful about using the proper method for different browsers

Reply:
>  Playwright ... hides it in chrome/chromium. 

Does it really though? It might hide it for Chromium, but it definitely did not hide it for Chrome the last time I tested it.

Reply:
I would like webdriver to be false with the default SDK configuration. Without any extra steps needed for the user. Otherwise, I don't care that much how it works 😄